docs/content/components/header.mdx

```diff
@@ -61,58 +61,48 @@ https://designsystem.digital.gov/components/header/
 			<ul class="nci-header-nav__primary">
 				<li class="nci-header-nav__primary-item">
 					<a
-						href="/"
-						class="nci-header-nav__primary-button"
-						aria-expanded="false"
-						aria-controls="megamenu-layer"
+						href="https://mocki.io/v1/f561a18e-d1aa-4c20-bde6-52099ad91502"
+						class="nci-header-nav__primary-link"
 					>
-						<span>Current section</span>
+						Current section
 					</a>
 				</li>
 				<li class="nci-header-nav__primary-item">
 					<a
-						href="/"
-						class="nci-header-nav__primary-button"
-						aria-expanded="false"
-						aria-controls="megamenu-layer"
+						href="https://mocki.io/v1/f561a18e-d1aa-4c20-bde6-52099ad91502"
+						class="nci-header-nav__primary-link"
 					>
-						<span>Second Section</span>
+						Second Section
 					</a>
 				</li>
 				<li class="nci-header-nav__primary-item">
 					<a
-						href="/"
-						class="nci-header-nav__primary-button"
-						aria-expanded="false"
-						aria-controls="megamenu-layer"
+						href="https://mocki.io/v1/f561a18e-d1aa-4c20-bde6-52099ad91502"
+						class="nci-header-nav__primary-link"
 					>
-						<span>Third Section</span>
+						Third Section
 					</a>
 				</li>
 				<li class="nci-header-nav__primary-item">
 					<a
-						href="/"
-						class="nci-header-nav__primary-button"
-						aria-expanded="false"
-						aria-controls="megamenu-layer"
+						href="https://mocki.io/v1/f561a18e-d1aa-4c20-bde6-52099ad91502"
+						class="nci-header-nav__primary-link"
 					>
-						<span>Fourth Section</span>
+						Fourth Section
 					</a>
 				</li>
 				<li class="nci-header-nav__primary-item">
 					<a
-						href="/"
-						class="nci-header-nav__primary-button"
-						aria-expanded="false"
-						aria-controls="megamenu-layer"
+						href="https://mocki.io/v1/f561a18e-d1aa-4c20-bde6-52099ad91502"
+						class="nci-header-nav__primary-link"
 					>
-						<span>Fifth Section</span>
+						Fifth Section
 					</a>
 				</li>
 			</ul>
 			<button class="nci-header-mobilenav__open-btn">Menu</button>
 			<div class="nci-header-nav__secondary">
-				<form class="nci-header-search" role="search">
+				<form class="nci-header-search" aria-label="sitewide" role="search">
```

Comment:
Can you add `action="#` and `method="get"` here

Also add some developer usage to the documentation on form action and also the input `name="search"`

We should also consider adding something to accessibility documentation about why `aria-label="sitewide"` is important if the site has multiple types of searches.